GOVERNANCE.md

```diff
@@ -0,0 +1,158 @@
+
+Team members are added to the [GitHub organization][gh] and have commit rights to all projects. They should however respect the maintainers of each project.
+
+Team members may retire at any time by emailing [the team][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team]. For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+
+Upon death of a member, their team membership ends automatically.
+
+Unless voted upon otherwise, membership changes take effect immediately. Unless voted upon otherwise, any membership change is announced to [the public developer mailing list][devs] as soon as it is in effect.
+
+### Maintainers
+
+Maintainers are team members who are lead one or more project(s) and serve as a point of conflict resolution amongst the contributors to this project.
+
+A maintainer or committer may resign by notifying the [team mailing list][team]. A maintainer with no project activity for a year is considered to have resigned. Maintainers who wish to resign are encouraged to propose another team member to take over the project.
+
+An extraordinary change of maintainership of any project may be proposed on the [team mailing list][team] and is voted on by [consensus](#consensus). If no consensus can be reached, the matter is resolved by [majority vote](#majority-vote).
+
+A project may have more multiple maintainers, as long as the responsibilities are clearly agreed upon between them. This includes coordinating who handles which issues and pull requests.
+
+Any maintainership change is performed by changing the project's MAINTAINERS.md on the main branch of the repository, and should be announced on the [developers mailing list][devs].
+
+### Technical decisions
+
+Technical decisions that only affect a single project are made informally by the maintainer of this project, and [lazy consensus](#consensus) is assumed.
+
+Technical decisions that span multiple parts of the Prometheus Project are discussed and made on the [Prometheus developer mailing list][devs]. Decisions are usually made by [consensus](#consensus). If no consensus can be reached, the matter may be resolved by [majority vote](#majority-vote).
+
+### New projects
+
+New projects under the Prometheus Project must be proposed to the [developer mailing list][devs] and are subject to [lazy consensus](#consensus).
+
+### Governance changes
+
+Changes to this document are discussed publicly on the [developer mailing mailing list][devs]. Any change requires a [supermajority](#supermajority-vote) in favour.
+
+### Other matters
+
+Any matter that needs a decision, including but not limited to financial matters, may be called to a vote by any member if they deem it necessary. For private or personal matters discussion and voting takes place on the [team mailing list][team], otherwise on the [developer mailing list][devs].
```

Comment:
I added this as a catch-all. Financial matters were listed before, so I included them here, but there's probably something we haven't thought of yet.

Reply:
Does "private" include normal financial matters? Not sure we'll want to talk financials too publicly... not that we currently have any financials (I assume PromCon is separate from this governance)

Reply:
"personal" means personal matters of the team, e.g. health and marital status. I don't think that's the word we want. Personnel is probably what you meant.

Reply:
I think it would not be outlandish to treat PromCon as a project under this, but it also doesn't need to be. I'll add financial to the list, and change personal -> personnel. in the end, "private matters" is wide enough to catch anything we deem "should not discuss publicly".

Reply:
Not that I'm against PromCon being community-owned, but it may (at least in the future) not be exactly the same community team as Prometheus (dev) team members, and it already has a separate organizers list and the domain is still owned by me. So I would at least want to think about that a bit further before automatically covering it under this governance.

Reply:
makes sense!